linkedlist.mod/doc/tlist_sort.bmx

```diff
@@ -0,0 +1,51 @@
+SuperStrict
+
+' create a list to hold some objects
+Local list:TList = new TList
+
+' add some string objects to the end of the list
+list.AddLast("short")
+list.AddLast("longer")
+list.AddLast("the longest")
+
+
+' DEFAULT SORT
+' sort them (in this case this leads to an alphabetic sort)
+' second parameter sets sort to ascending or not
+list.Sort(True)
+
+' enumerate all the strings in the list
+For Local a:String = EachIn list
+	Print a
+Next
+
+' outputs:
+' longer
+' short
+' the longest
+
+
+' CUSTOM SORT
+' define a custom compare function
+Function MyCompare:int( o1:Object, o2:Object )
+	if len(string(o1)) < len(string(o2))
```

Comment:
Will adjust to your comments.

For years I taught myself to use `then` in single line and not in multi line.
Is there a 'rule'/coding standard somewhere so we have the examples following it?

Reply:
While I was on my way to append the `Then` to the `If`-cases I've read the documentation to "if", "elseif", "then" etc.
The documentation gives most examples without `Then`. Maybe we should consider adding `Then` there too (for consistency).

Reply:
Regarding `Then` again.
https://blitzmax.org/docs/en/tutorials/beginners_guide/

> When to use Then? You can put it after your If expression. The use of Then, is not required and I think the code is just as easy to read without Then. Like this, If A = 1 Then B = 2 is equal to If A = 1 B = 2. If you take a BlitzMax file and delete all Then, it will run the same. Use Then if you think it helps you read the code. When I use then, then it is to make an ifstatement on one line more readable, like in the example above and in the example below. My advice is, never use then if your if-statement is not on one line.

Will open up an issue for the site-repo (there is other stuff no longer being correct)